mathicsscript/termshell.py

```diff
@@ -278,7 +248,7 @@ def read_line(self, prompt):
             line = input(prompt)
         if line.startswith("!") and self.lineno == 0:
             raise ShellEscapeException(line)
-        return replace_unicode_to_wl(line)
+        return replace_unicode_with_wl(line)
```

Comment:
Where is `replace_unicode_with_wl` imported in this module?

Reply:
I forgot it 😁️